packages/webdriver/src/utils.js

```diff
@@ -382,3 +396,35 @@ export function setupDirectConnect(params) {
         params.path = directConnectPath
     }
 }
+
+/**
+ * get human readable message from response error
+ * @param {Error} err response error
+ */
+export const getSessionError = (err) => {
+    // browser driver / service is not started
+    if (err.code === 'ECONNREFUSED') {
+        return `Unable to connect to "${err.address}:${err.port}", make sure browser driver is running on that address.`
+    }
+
+    if (!err.message) {
+        return 'See logs for more information.'
+    }
+
+    // wrong path: selenium-standalone
+    if (err.message.includes('Whoops! The URL specified routes to this help page.')) {
+        return "Make sure there is no `path: '/'` or path set to valid uri, ex: '/wd/hub', in wdio.conf!"
```

Comment:
How about:
```suggestion
        return "It seems you are running a Selenium Standalone server and point to a wrong path. Please set `path: '/wd/hub'` in your wdio.conf.js!"
```

Reply:
we could also try to fix the path automatically, maybe in a different PR

Reply:
Agree, depending on service used. Actually we can do it in the services!

Reply:
I'll change in selenium standalone and raise pr or issue to chrome and firefox services